docs/src/pages/getting-started/support/support.md

```diff
@@ -63,44 +63,7 @@ The current status of each Material-UI version is as follows:
 
 For teams and organizations that require additional support for older versions, Material-UI has [options available](#enterprise).
 
-## Professional support (Premium)
-
-### Enterprise Edition
```

Comment:
I have mixed feelings about this one. I think that with more effort, we might be able to yield something interesting from it. Version migration assistance will rise with the v5 effort. I think that we can remove it and put it back online as we grow the team. I think before we put the advisory back online. I will try to invest time into this item, happy to remove it for now.